docs/extensions/semgrep-intellij.md

```diff
@@ -0,0 +1,102 @@
+---
+slug: semgrep-intellij
+append_help_link: true
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Semgrep IntelliJ extension
+
+[Semgrep](https://semgrep.dev/) accelerates your security journey by swiftly scanning code and package dependencies for known issues, software vulnerabilities, and detected secrets with unparalleled efficiency. By default, the Semgrep IntelliJ extension scans code whenever you change or open files.
+
+:::info Prerequisites
+
+The Semgrep IntelliJ extension communicates with Semgrep command-line interface (CLI) to run scans. Install Semgrep CLI before you can use the extension. To install Semgrep CLI:
+
+```sh
+# For macOS
+$ brew install semgrep
+
+# For Ubuntu/WSL/Linux/macOS
+$ python3 -m pip install semgrep
+
+# To try Semgrep without installation run via Docker
+$ docker run --rm -v "${PWD}:/src" returntocorp/semgrep semgrep
+```
+:::
+
+## Quick start
+
+1. Install the Semgrep extension:
+   -  Visit [Semgrep's page on the Jet Brains Marketplace](https://plugins.jetbrains.com/plugin/22622-semgrep).
+   -  In IntelliJ: **Settings/Preferences > Plugins > Marketplace > Search for `semgrep-intellij` > Install**. You may need to restart IntelliJ for the Semgrep extension to be installed.
+
+2. Sign in: Press <kbd>Ctrl+⇧Shift+A</kbd> (Windows) or <kbd>⌘Command+⇧Shift+A</kbd> (macOS) and sign in to Semgrep Cloud Platform by selecting the following command:
+   ```
+   Sign in with Semgrep
+   ```
+3. Test the extension by pressing <kbd>Ctrl+⇧Shift+A</kbd> (Windows) or <kbd>⌘Command+⇧Shift+A</kbd> (macOS) and run the following command:
+   ```
+   Scan workspace with Semgrep
+   ```
+4. See Semgrep findings: Hold the pointer over the code that has the red underline.
```

Comment:
Does the "hover over a line to see more information about the finding" work for IntelliJ?

Reply:
@milanwilliams is SME for this, so I'll defer to her, but I assumed so as she wrote it.

Reply:
yea it should, line can also be different colors (highlighted yellow in intellij)